mariaDB4j-core/src/main/java/ch/vorburger/mariadb4j/springframework/MariaDB4jSpringService.java

```diff
@@ -21,6 +21,7 @@
 
 import ch.vorburger.exec.ManagedProcessException;
 import ch.vorburger.mariadb4j.MariaDB4jService;
+import com.google.common.base.Objects;
```

Comment:
```suggestion
import import java.util.Objects;
```

> @qxo unfortunately [this still fails to build on CI](https://github.com/vorburger/MariaDB4j/actions/runs/4338690720/jobs/7581625339), and therefore cannot be merged.

@qxo would you like to fix the build on this PR so that I can merge this? It fails because you are using `import com.google.common.base.Objects;` but this project does not have a dependency on the Google Guava library (and should not have one). Could you change it to use `import java.util.Objects;` instead please?